lib/sanity.js

```diff
@@ -67,6 +67,33 @@ function createSanity(options) {
       await execa('git', ['commit', '-m', 'Initial Sanity commit'], {
         cwd: this.projectDir
       });
+
+      await execa(
+        'sanity',
+        ['cors', 'add', 'http://localhost:3000', '--no-credentials'],
+        {cwd: this.projectDir}
```

Comment:
Does it matter which directory these commands need to be run in? You may need `this.sanityDir` instead